sky/sdk/lib/widgets/scrollable.dart

```diff
@@ -155,6 +179,8 @@ abstract class Scrollable extends StatefulComponent {
     double eventVelocity = direction == ScrollDirection.horizontal
       ? -event.velocityX
       : -event.velocityY;
+    if (_scrollToPerformance.isAnimating)
+      _scrollToPerformance.stop();
```

Comment:
We need to do this in _handlePointerDown too.  Maybe move this work into _stopSimulation?  Notice that _startSimulation calls _stopSimulation.  We might want to rename _stopSimulation to _stop() ?